test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,55 @@
+const { StatusCodes } = require('http-status-codes');
+const chai = require('chai');
+const { expect } = require('chai');
+const chaiSubset = require('chai-subset');
+const md5 = require('md5');
+
+chai.use(chaiSubset);
+const axios = require('axios');
+
+const urlBase = 'https://api.github.com';
+const repoName = 'jasmine-json-report';
+
+describe('Consuming GET Method GitHub API', () => {
+  it('Consume GET Service', async () => {
+    const response = await axios.get(`${urlBase}/users/aperdomob`);
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.name).to.eql('Alejandro Perdomo');
+    expect(response.data.company).to.eql('Perficient Latam');
+    expect(response.data.location).to.eql('Colombia');
+  });
+
+  it('Repo List Aperdomob', async () => {
+    const repos = await axios.get(`${urlBase}/users/aperdomob/repos`);
+    const repo = repos.data.find((e) => e.name === repoName);
+    expect(repos.data[12].name).to.eql(repo.name);
+    expect(repo.full_name).to.eql('aperdomob/jasmine-json-report');
+    expect(repo.private).to.eql(false);
+    expect(repo.description).to.eql('A Simple Jasmine JSON Report');
+  });
+
+  it('Downloading jasmine-json-report Repository', async () => {
+    const response = await axios.get(`${urlBase}/repos/aperdomob/${repoName}/zipball/master`);
```

Comment:
the repository response of the previous test (which you store in the repo variable) can be used to get this URL (if you use `repo.url`):
![image](https://user-images.githubusercontent.com/10209180/176711611-92f3d2a4-def3-44be-8bbf-837f36507eb0.png)